README.md

```diff
@@ -106,38 +106,23 @@ A collection of demos utilizing trestle can be found in the related project [com
 
 Compliance trestle is currently stable and is based on NIST OSCAL version 1.0.4, with active development continuing.
 
-## Community call
+## Community meetings and communications
 
-We would like to share development in progress for compliance trestle, coming soon and get feedback from community on what features would they like to see in compliance trestle.\
-The community call will happen every 2 week(s) on Tuesday at 10.00am EST.\
-Meeting information:
+##### Scheduled meetings
 
-```
-Compliance Trestle Community Call
-
-Join from the meeting link
-https://ibm.webex.com/ibm/j.php?MTID=m030fdef5ac2d09f46f04813bb5e9dc6b
-Tuesday, September 5, 2023 10:00 AM | 30 minutes | (UTC-04:00) Eastern Time (US & Canada)
-Occurs every 2 week(s) on Tuesday effective 9/5/2023 from 10:00 AM to 10:30 AM, (UTC-04:00) Eastern Time (US & Canada)
- 
-Join by meeting number
-Meeting number (access code): 146 967 4515
- 
-Tap to join from a mobile device (attendees only)
-1-844-531-0958,,1469674515#43533276# United States Toll Free
-+1-669-234-1178,,1469674515#43533276# United States Toll
-Some mobile devices may ask attendees to enter a numeric password.
- 
-Join by phone
-1-844-531-0958 United States Toll Free
-1-669-234-1178 United States Toll
-Global call-in numbers  |  Toll-free calling restrictions
- 
-Join from a video system or application
-Dial 1469674515@ibm.webex.com
-You can also dial 173.243.2.68 and enter your meeting number.
+Please attend! All are invited.
 
-```
+When: 2nd and 4th Tuesday of each month at 10:00 ET [convert to your local time](https://dateful.com/convert/est-edt-eastern-time)
+
+Where: [https://zoom.us/j/92729235315](https://zoom.us/j/92729235315)
+
+- Meeting Id: 927 2923 5315
+
+- Meeting agenda and notes: [here](https://docs.google.com/document/d/1z9xvt-Z97j4CtEH1-nR9sMWul7jQkUi_fNY7BdMPgxM/edit#heading=h.nohkp1kbeduj)
+
+##### Chat anytime
```

Comment:
This should not block this particular change, but since the Slack and Zoom meeting are for the `oscal-compass` organization, I wonder if this type of information would belong in an organization profile or in a community repository.

Reply:
Where would this go, when I visit https://github.com/oscal-compass? Also, replicating information seemed doomed to inconsistency, so perhaps one should point to the other.

Reply:
We can have an organizational profile [READMEs](https://docs.github.com/en/organizations/collaborating-with-groups-in-organizations/customizing-your-organizations-profile) so the information would be available when the organization link is visited. Example here for the CNCF org - https://github.com/cncf/.github/tree/master/profile. Totally agree that we should not duplicate the information, but link to it.

Reply:
So I think you mean here https://github.com/oscal-compass/.github in out case. Seems fine to me, but then perhaps we should open an issue against that repo and make a proposal as to what it should contain, pre-CNCF.

Reply:
Added here https://github.com/oscal-compass/.github/issues/1